nilearn/plotting/tests/test_html_stat_map.py

```diff
@@ -69,23 +69,29 @@ def test_threshold_data():
     data = np.arange(-3, 4)
 
     # Check that an 'auto' threshold leaves at least one element
-    data_t, thresh = html_stat_map._threshold_data(data, threshold='auto')
```

Comment:
could we also check the returned threshold and data?

Reply:
Sorry, I don't think I understand this one. Just checking the value of the generated threshold ?